discord_music_bot/commands/audio.py

```diff
@@ -0,0 +1,171 @@
+import re
+from datetime import timedelta
+
+import discord
+import wavelink
+
+from discord_music_bot.commands.base import BaseCog
+from discord_music_bot.helpers import format_timedelta, get_current_player
+
+
+class AudioCommands(BaseCog):
+    @discord.slash_command(
+        name="pause", description="pause the currently playing song"
+    )
+    async def pause(self, ctx: discord.ApplicationContext) -> None:
+        player = await get_current_player(ctx)
+
+        if not player.is_playing():
+            await ctx.respond("Not playing")
+            return
+
+        await player.pause()
+        await ctx.respond("Paused playback")
+
+    @discord.slash_command(
+        name="resume", description="resume playing the currently paused song"
+    )
+    async def resume(self, ctx: discord.ApplicationContext) -> None:
+        player = await get_current_player(ctx)
+
+        if not player.is_paused():
+            await ctx.respond("Not paused")
+            return
+
+        await player.resume()
+        await ctx.respond("Resuming playback")
+
+    @discord.slash_command(
+        name="shut_the_fuck_up", description="SHUT THE FUCK UP"
+    )
+    async def leave(self, ctx: discord.ApplicationContext) -> None:
+        player = await get_current_player(ctx)
+
+        await player.disconnect()
+        await ctx.respond("Ok")
+
+    @discord.slash_command(name="volume", description="set audio volume")
+    @discord.option(
+        "volume",
+        int,
+        min_value=0,
+        max_value=1000,
+        description="number from 0 to 1000",
```

Comment:
Fixed the description. There is no default value because if I provide one, the option will be optional. I think it makes little sense if the `volume` option is not required in `/volume` command.